content/03.methods.md

```diff
@@ -184,4 +192,7 @@ We used Pubmed Central's DOI converter [@url:https://www.ncbi.nlm.nih.gov/pmc/pm
 We retained articles that were included in the PMCOA corpus and performed a token analysis as described to compare these preprints with their published versions.
 As above, we generated document embeddings for every obtained preprint and published article.
 We projected these preprint embeddings onto our publication landscape to visually observe the dispersion of this subset.
-Finally, we performed a time analysis that paralleled our approach for the full set of preprint-publication pairs to examine relationships between linguistic changes and the time to publication.   
+We performed a time analysis that paralleled our approach for the full set of preprint-publication pairs to examine relationships between linguistic changes and the time to publication.
+The longest publication time within the "Preprints in Motion" subset was 195 days; however, our bioRxiv snapshot contains preprint-published pairs with publication times longer than this timepoint.
+We performed a secondary analysis that accounts for these publication pairs.
```

Comment:
```suggestion
The optimum comparison for this set would likely be to consider only preprints posted on the same days as "Preprints in Motion" ones, but, based on our results examining publication rate over time, these may not yet have made it fully through the publication processes.
We performed a secondary analysis to control for the time since posting in which we filtered the bioRxiv snapshot to only contain publication pairs with publication time of less than or equal to 195 days.
```